templates/frontend/components/profile_activity.html.twig

```diff
@@ -0,0 +1,53 @@
+{% block profile_activity %}
+    <div class="col-xs-12 col-md-8">
+        <div class="card">
+            <div class="card-title">
+                {{ 'profile.activity.activity'|trans }}
+            </div>
+            {% for comment in comments %}
+                <div class="card-body">
```

Comment:
this should be in a `<ul class="card-body">` or `<div class="card-body"><ul>...`
and each comment should be a `<li>`